account_financial_report/report/aged_partner_balance.py

```diff
@@ -103,19 +103,7 @@ def _get_move_lines_data(
         domain = self._get_move_lines_domain_not_reconciled(
             company_id, account_ids, partner_ids, only_posted_moves, date_from
         )
-        ml_fields = [
-            "id",
-            "name",
-            "date",
-            "move_id",
-            "journal_id",
-            "account_id",
-            "partner_id",
-            "amount_residual",
-            "date_maturity",
-            "ref",
-            "reconciled",
-        ]
+        ml_fields = self._get_ml_fields()
```

Comment:
Instead of putting all fields again, please put the common ones in abstract, and here only the extra ones.

Reply:
Hi @pedrobaeza I just did that :) Except for the vat report, because it has no common fields, just id field.